client/pages/viewEmail.js

```diff
@@ -0,0 +1,152 @@
+import Link from "next/link";
+import React, { useState, useEffect } from "react";
+import CustomButton from "./components/CustomButton";
+import PageTitle from "./components/PageTitle";
+import TextBox from "./components/TextBox";
+
+export default function () {
+  const [email, setEmail] = useState(null); // [value, function(){sets value and re-renders}]
+
+  useEffect(() => {
```

Comment:
Todo finish with a request and add changes depending on if it’s an inbox item or a sent item.